donation-tracker/src/pages/404.tsx

```diff
@@ -1,23 +1,27 @@
 import * as React from 'react';
-import { Link } from 'gatsby';
-import TestComponent from '../components/testComponent';
+import LayoutModule from '../components/layout';
+import PageMetadata from '../components/pageMetadata';
+import { Card, CardContent, CardHeader, Container, Typography } from '@mui/material';
 
 // markup
 const NotFoundPage = () => {
   return (
-    <main>
-      <title>Not found</title>
-      <h1>Page not found</h1>
-      <p>
-        Sorry{" "}
-        <span role="img" aria-label="Pensive emoji">
-          😔
-        </span>{" "}
-        we couldn’t find what you were looking for.
-        <br />
-        <Link to="/">Go home</Link>.
-      </p>
-    </main>
+    <LayoutModule>
+      <PageMetadata title="Page not found / Seite nicht gefunden"></PageMetadata>
+      &nbsp;
```

Comment:
Ahhh, stop using those 😂
Maybe simply add a `mt={2}` or so to the `<Container>` below. Maybe next time... pleeeeease 😋

Reply:
OK... 😔